app/components/Login/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Button, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
+
+import styles from './styles.css';
+
+const Login = (props) => {
+
+  const { email, password } = props;
+
+  return (
+    <div className={styles.loginForm}>
+      <Form>
+        <Label htmlFor='username'>email</Label>
+        <Input
+          name='username'
+          type='email'
+          placeholder='person@example.com'
+          value={email}
+          onChange={(e) => props.changeUserEmail(e.target.value)}
```

Comment:
i would personally add `changeUserEmail` to the props deconstructor, i.e.

`const { email, password, changeUserEmail } = props;`